libs/pmp-web/auth/shell/src/lib/shell-routing.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { LoginComponent } from './containers/login/login.component';
+
+const routes: Routes = [
+  {
+    path: 'auth',
+    component: LoginComponent,
+    children: [
+      {
+        path: '',
+        loadChildren: () =>
+          import('@pimp-my-pr/pmp-web/auth/login-feature').then(m => m.PmpWebAuthLoginFeatureModule)
+      }
+    ]
```

Comment:
I'm sure that here should appear 'login' path but I'm not sure if 'auth' is necessary.